Source/WebKit/WebProcess/WebPage/CoordinatedGraphics/DrawingAreaCoordinatedGraphics.cpp

```diff
@@ -465,7 +465,7 @@ void DrawingAreaCoordinatedGraphics::targetRefreshRateDidChange(unsigned rate)
 {
     UNUSED_PARAM(rate);
 #if !USE(GRAPHICS_LAYER_TEXTURE_MAPPER)
-    if (m_layerTreeHost)
```

Comment:
I just found out this PR and I have a similar situation (using wpe and cog master) -- https://bugs.webkit.org/show_bug.cgi?id=242593

But I have to say, this doesn't look like a proper fix to me, but rather just way to dodge the assertion.

Reply:
Yeah, not really sure what the correct fix is for this, there's a second issue with calls happening on unexpected threads in #2329 that I'm seeing as well.